guides/v2.2/magento-functional-testing-framework/release-2/introduction.md

```diff
@@ -56,9 +62,9 @@ As a software engineer, perform regression testing before release to ensure that
 
 ## Find your MFTF version
 
-The MFTF is installed as a Composer dependency in `<magento_root>/app/code/<vendor_name>/<module_name>/Test/Mftf/composer.lock`.
+The MFTF is installed as a Composer dependency in `<magento_root>/app/code/dev/Tests/acceptance/` directory.
```

Comment:
MFTF 2.2.0 path is actually `<magento_root>/dev/tests/acceptance/`, unless this line is supposed to refer to test packages composer.json (which I don't think it is).